### PR TITLE
feat: migrate kras and dar to mainnet and testnet

### PR DESCRIPTION
**Pull request by shamsartem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +37,7 @@ curl -qsL https://raw.githubusercontent.com/fluencelabs/cli/main/install.sh | ba
 
 ### Update
 
-To select `fluence` version compatible with certain network (`dar`, `stage`,
-`kras`) run:
+To select `fluence` version compatible with certain network (`testnet`, `stage`, `mainnet`) run:
```
Comment (folex) [inline README.md:40]:
```suggestion
To select `fluence` version compatible with certain network (`stage`, `testnet`, `mainnet`) run:
```

File: cli/docs/configs/provider.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Logs configuration
 
 | Property         | Type    | Required | Description                     |
 |------------------|---------|----------|---------------------------------|
-| `logLevel`       | string  | No       | Log level. Default: info        |
+| `logLevel`       | string  | No       | Log level. Default: debug       |
```
Comment (folex) [inline cli/docs/configs/provider.md:61]: Hm, will it always show debug logs now?

Comment (shamsartem) [reply cli/docs/configs/provider.md:61]: @justprosh asked for this to be the default

Comment (justprosh) [reply cli/docs/configs/provider.md:61]: yeah, for first phases of mainnet/testnet i want to see all logs by default in providers' grafanas

**Review by folex — APPROVED** [review]

File: cli/docs/configs/provider.md (hunk 2 above)
Comment (folex) [reply cli/docs/configs/provider.md:61]: Even for these?

```
nox::bootstrapper
yamux::connection::stream
tokio_threadpool
tokio_reactor
mio
tokio_io
soketto
yamix
multistream_select
libp2p_swarm
libp2p_secio
libp2p_websocket::framed
libp2p_ping
libp2p_core::upgrade::apply
libp2p_core::upgrade
libp2p_kad::kbucket
libp2p_kad
libp2p_kad::query
libp2p_kad::iterlog
libp2p_plaintext
libp2p_identify::protocol
wasmer_wasi
wasmer_interface_types_fl
polling
walrus
regalloc2
cranelift_wasm
wasmtime_cranelift
tokio
libp2p_noise
yamux
wasmtime_jit
wasi_common
particle_reap
marine_core::module::marine_module
runtime::resource
```

**Pull request merged**